tests/test_uncertain_conditional.py

```diff
@@ -237,21 +222,15 @@ def test_quadrature(white, mean):
     effective_mean = mean_function or (lambda X: 0.0)
 
     def conditional_fn(X):
-        return conditional(X,
-                           inducing_variable,
-                           kernel,
-                           DataQuad.q_mu,
-                           q_sqrt=DataQuad.q_sqrt,
-                           white=white)
+        return conditional(X, inducing_variable, kernel, DataQuad.q_mu, q_sqrt=DataQuad.q_sqrt, white=white)
 
     def mean_fn(X):
         return conditional_fn(X)[0] + effective_mean(X)
 
     def var_fn(X):
         return conditional_fn(X)[1]
 
-    quad_args = DataQuad.Xmu, DataQuad.Xvar, DataQuad.H, DataQuad.D_in, (
-        DataQuad.D_out, )
+    quad_args = DataQuad.Xmu, DataQuad.Xvar, DataQuad.H, DataQuad.D_in, (DataQuad.D_out, )
```

Comment:
```suggestion
    quad_args = DataQuad.Xmu, DataQuad.Xvar, DataQuad.H, DataQuad.D_in, (DataQuad.D_out,)
```
PEP-8

Reply:
(actually there's a bunch more cases around the codebase that have the same pattern)

Reply:
> Avoid extraneous whitespace in the following situations: [...] Between a trailing comma and a following close parenthesis.

(ref: https://www.python.org/dev/peps/pep-0008/#pet-peeves)